Unable to build library test project in Visual Studio

### Description

Hello 👋! 

I'm trying to work on `Microsoft.Extensions.Logging.Console` in Visual Studio _Preview_.

I can build and run the tests successfully from the command line, but when I open the library project in VS with
```.\build.cmd -vs  Microsoft.Extensions.Logging.Console```
I am unable to build the test projects due to the following error:
```
2>------ Build started: Project: Microsoft.Extensions.Logging.Console.Tests, Configuration: Debug Any CPU ------
2>C:\Program Files (x86)\Microsoft Visual Studio\2019\Preview\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(4469,5): error MSB4006: There is a circular dependency in the target dependency graph involving target "_GetCopyToOutputDirectoryItemsFromTransitiveProjectReferences".
2>Done building project "TestUtilities.csproj" -- FAILED.
2>C:\Program Files (x86)\Microsoft Visual Studio\2019\Preview\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(4469,5): error MSB4006: There is a circular dependency in the target dependency graph involving target "_GetCopyToOutputDirectoryItemsFromTransitiveProjectReferences".
2>Done building project "TestUtilities.csproj" -- FAILED.
```
Any idea of what I could be doing wrong? Thank you!

<!--
* Please share a clear and concise description of the problem.
* Include minimal steps to reproduce the problem if possible. E.g.: the smallest possible code snippet; or a small repo to clone, with steps to run it.
* What behavior are you seeing, and what behavior would you expect?
  -->

### Configuration

Visual Studio: Version 16.7.0 Preview 6.0
Windows Pro x64 Version 10.0.19041 Build 19041

master @ 065bf96e5c2

### Regression?

build problem

--- a/src/libraries/Common/tests/TestUtilities/TestUtilities.csproj
+++ b/src/libraries/Common/tests/TestUtilities/TestUtilities.csproj
@@ -1,6 +1,8 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <!-- For some reason, xunit.core.props isn't excluded in VS and sets this to true. -->
+    <IsTestProject>false</IsTestProject>
     <!--
       This assembly is referenced from rid agnostic configurations therefore we can't make it RID specific
       and instead use runtime checks.

